code/__DEFINES/misc.dm

```diff
@@ -406,3 +406,9 @@ GLOBAL_LIST_INIT(ghost_others_options, list(GHOST_OTHERS_SIMPLE, GHOST_OTHERS_DE
 #define NO_FIELD 0
 #define FIELD_TURF 1
 #define FIELD_EDGE 2
+
+//gibtonite state defines
+#define GIBTONITE_UNSTRUCK 0
+#define GIBTONITE_ACTIVE 1
+#define GIBTONITE_INERT 2
```

Comment:
Could you use STABLE or RECLAIMED here? Gibtonite doesn't become harmless; it just waits.